### ISSUE TITLE
Internal Server Error after every prompt

### ISSUE THREAD

**Issue by idoll** [stackblitz/bolt.new#2271]
### Describe the bug

Every prompt I write across 3 different apps results in an "Internal Server Error"

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-3nhp3p

### Steps to reproduce

Type a prompt....get internal server error

### Expected behavior

No internal server error

### Screen Recording / Screenshot

![image](https://github.com/user-attachments/assets/e01287b6-2633-49ec-86ff-cd38b4d3fd40)

### Platform

Windows, Chrome, 130.0.6723.117

**Comment by Frankg40** [comment]
We're all getting it.

Theyll fix it shortly im sure

**Comment by TryLovingMichael** [comment]
Yeah same issue

**Comment by SecteurA** [comment]
Yeah they are upgrading something again, yesterday same time i had issues and i got a response : Hi [@SecteurA](https://github.com/SecteurA),

Given the high demand we have been seeing the team was making some database upgrades but everything should be good to go now! We will work to communicate this more clearly in the future. Appreciate your patience as improvements continue to be made!

**Comment by KazeroG** [comment]
duplicates #2270
